problems/3.conditionals.js

```diff
@@ -2,15 +2,25 @@
 * @param {number} temp - The current temperature
 * @returns {string} - A message about the temperature
 */
-function temperatureMessage() {
+function temperatureMessage(temp) {
+  if(temp < 32) {
+    return "It's freezing!"
+  }else if(temp > 32) {
+    return "It's alright"
+  }else {
+    return "It's alright"
+  }
 
 }
 
 /** Returns "It's freezing!" if the temperature is below 32, and "It's alright" otherwise. The implementation must use a ternary.
 * @param {number} temp - The current temperatur
 * @returns {string} - A message about the temperature
 */
-function temperatureMessageUsingTernary() {
+function temperatureMessageUsingTernary(temp) {
+   let (temp <= 32);{
```

Comment:
incorrect syntax. Please review. let is variable declaration. 
You need a condition. 
return tem <= 32 ? "freezing" : "alright"